src/btree/bt_compact.c

```diff
@@ -135,8 +142,11 @@ __wt_compact(WT_SESSION_IMPL *session)
      * source if we make no progress, set a flag if the block layer thinks compaction is possible.
      */
     WT_RET(bm->compact_skip(bm, session, &skip));
-    if (skip)
+    if (skip) {
+        WT_STAT_CONN_INCR(session, session_table_compact_skipped);
+        WT_STAT_DATA_INCR(session, btree_compact_skipped);
```

Comment:
Why do we need to have both `session_table_compact_skipped` and `btree_compact_skipped` if they have the same value?